4.3.0 Getting started enhancement Stage 1 (#5002)
wazuh/wazuh-documentation: source/getting-started/use-cases/regulatory-compliance.rst

Regulatory compliance
=====================

> The Wazuh platform is often used to meet the technical aspects of regulatory compliance standards. Wazuh uses its SIEM capabilities to centralize, analyze, and enrich security data and provide security controls, such as intrusion detection, configuration assessment, log analysis, and vulnerability detection, to meet compliance requirements.

Reviewer: ```suggestion
Wazuh uses its SIEM capabilities to centralize, analyze and enrich security data. In addition, it provides security controls, such as intrusion detection, configuration assessment, log analysis, and vulnerability detection, to meet the technical aspects of regulatory compliance standards.
```
